snapcraft/plugins/kbuild.py

```diff
@@ -0,0 +1,179 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""The kbuild plugin is used for building kbuild based projects as snapcraft
+parts.
+
+This plugin is based on the snapcraft.BasePlugin and supports the properties
+provided by that plus the following kbuild specific options with semantics as
+explained above:
+
+    - kdefconfig:
+      (list of kdefconfigs)
+      defconfig target to use to see base configuration. default: "defconfig"
```

Comment:
this is not super clear. "to use to see" sounds bad, but I'm not sure how to fix it because I don't understand what it means "to see". Do you have any possible rewrite in mind?